sai_p4/instantiations/google/tor.p4

```diff
@@ -62,8 +63,8 @@ control egress(inout headers_t headers,
 #endif
 @pkginfo(
   name = PKG_INFO_NAME,
-  organization = "Google",
-  version = SAI_P4_PKGINFO_VERSION_HAS_PACKET_OUT_SUPPORT
+  organization = "Google"
+//TODO  version = SAI_P4_PKGINFO_VERSION_HAS_PACKET_OUT_SUPPORT
```

Comment:
Why is this commented out ? The TODO format is
// TODO(PINS): <one liner on why this is disabled>

Reply:
@kishanps , i have taken care of this version in the next PR - https://github.com/sonic-net/sonic-pins/pull/193